package.json

```diff
@@ -119,8 +118,8 @@
     "ts-loader": "^5.4.5",
     "ts-node": "^8.6.2",
     "typed-error": "^2.0.0",
-    "typescript": "^3.7.5",
-    "webpack": "^4.25.0",
+    "typescript": "^3.8.3",
+    "webpack": "^4.28.0",
```

Comment:
This is not actually updated, just setting the package.json version to match the package-lock.json